src/components/CodeEditor.js

```diff
@@ -140,10 +210,10 @@ class CodeEditor extends Component {
     // If it's a number,
     if (token && token.type === 'number') {
       // clear out the previous mark,
-      this.mark && this.mark.clear()
+      this.sliderMark && this.sliderMark.clear()
 
       // and mark this token.
-      this.mark = this.codeMirror.markText(
+      this.sliderMark = this.codeMirror.sliderMarkText(
```

Comment:
I agree with renaming `mark` to `sliderMark`, but we need to leave `this.codeMirror.markText` as is. This was probably a simple typo.

Reply:
ah yup my bad